Sources/SwiftCrossUI/Backend/AppBackend.swift

```diff
@@ -720,13 +806,25 @@ extension AppBackend {
     }
 }
 
+public protocol SheetImplementation {
+    var sheetSize: SIMD2<Int> { get }
+}
```

Comment:
Using a protocol here is nice, but I've generally tried to keep the backend API really consistent in its design. Unfortunately we can't use protocols for a lot of the widget-related stuff so I've gone with the decision of making all backend data types opaque and putting their getters and setters on the backend. It's not super Swifty, but the goal is to make backend implementation surprise-free, so it's kinda the best we can do with today's Swift.

tl;dr remove the protocol and move `sheetSize` to a `size(ofSheet:)` method on the  AppBackend protocol.

If there's a technical reason that it has to be done this way (e.g. to access sheet sizes in a place with no backend access or something), let me know